connectivity/manifests/client-egress-to-cidr-1111-knp.yaml

```diff
@@ -0,0 +1,16 @@
+# This policy allows packets towards {{.ExternalIP}}, but not {{.ExternalOtherIP}}.
+# Both addresses are owned by CloudFlare/APNIC.
```

Comment:
nit, I'm guessing this was copied from somewhere but if the IPs are templated then we don't know if they're CloudFlare/APNIC addresses.
```suggestion
```

Reply:
Fixed!